server/scheduling/scheduler.ts

```diff
@@ -0,0 +1,39 @@
+import { initManagers } from '../db/initDB';
+import { IPost } from '../db/models';
+import mongoose = require('mongoose');
+
+// eslint-disable-next-line no-unused-vars
+const { jobManager, postManager } = initManagers();
+const lt = require('long-timeout');
+
+// run all the tasks which are in the database
+(async() => {
+	(await jobManager.getAllJobs()).forEach((elem) => {
```

Comment:
The same question: the async function with the single await. This isn't clear what are we waiting for here?